docs/source/references/schema.json

```
@@ -1000,7 +1166,17 @@
       }
     ]
   },
-  "ExecutorEventSet": "u32",
+  "ExecutorEventSet": {
+    "Bitmap": {
+      "repr": "u32",
+      "masks": [
+        {
+          "name": "Upgraded",
+          "mask": 1
+        }
+      ]
+    }
+  },
```

```suggestion
  "ExecutorEventSet": {
    "Enum": [
      {
        "tag": "Upgraded",
        "discriminant": 1
      }
    ]
  },
```

why don't we just serialize it as an enum?